### ISSUE TITLE
Graph mouse-over is too busy

### ISSUE THREAD

**Issue by jake-flatsix** [pr3d4t0r/COVIDvu#184]
Mousing over one series displays data labels for all series. Can we limit that to just show the specific series?

Alternatively, build in a "filter" to the legend allowing the user to highlight a specific series (and/or mute the others).

![Screen Shot 2020-03-18 at 7 21 10 PM](https://user-images.githubusercontent.com/54414174/77025194-af242000-694d-11ea-95cb-f9ed875eba71.png)

**Comment by kinabalu** [comment]
clicking on a country in the legend will remove it from the series btw.

**Comment by jake-flatsix** [comment]
Nice! Didn't see that function...

**Comment by pr3d4t0r** [comment]
So...  is there an action item here?  Maybe a mouse over legend "click on label to make it disappear"?  Otherwise let's close the ticket.  @jake-flatsix - thoughts?

**Comment by kinabalu** [comment]
i think it's fine to keep this open, as we look at the design and UX of things, we can ask questions like the above on usability

**Comment by jake-flatsix** [comment]
Yeah, I’d keep it open at least until you nominate a UI/UX person (or assume that role!), and try a couple of different Plotly options. 

I may not have categorized the issue properly, so feel free to demote/deprioritize as you see fit. 

JM
